WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Exceptions/DevicePortalException.cs

```diff
@@ -31,6 +31,16 @@ namespace Microsoft.Tools.WindowsDevicePortal
 #endif // !WINDOWS_UWP
     public class DevicePortalException : Exception
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DevicePortalException"/> class.
+        /// </summary>
+        /// <param name="baseException">A base exception that we are converting to a DevicePortalException.</param>
+        public DevicePortalException(Exception baseException)
+        {
+            this.HResult = baseException.HResult;
```

Comment:
Consider checking if this is an Http exception and passing control to a method that knows how to decode. This will let us get rid of the other constructors we have.

Reply:
Note: We should probably mark the old constructors as deprecated before removing them, and then open a Breaking Change issue to remove them for v1